service/server_grpc.go

```diff
@@ -26,7 +26,6 @@ func parseUUID() grpctransport.ServerOption {
 
 func NewGRPCServer(endpoints KolideClient, logger log.Logger) pb.ApiServer {
 	options := []grpctransport.ServerOption{
-		grpctransport.ServerErrorLogger(logger),
```

Comment:
Note I killed this because the docs say
> ServerErrorLogger is used to log non-terminal errors. By default, no errors are logged.
and I think we should leave it at the default.

Reply:
I'm not sure I agree. This method is for server side implementations so it would not affect the launcher client. Is there a reason we wouldn't want to log if a grpc error happened (unable to decode, request failed etc)

IMO if you're removing it, the `NewGRPCServer` should accept a list of `grpctransport.ServerOption` because there's no way to add a logger if it's needed.